src/main/java/dinkplugin/notifiers/KillCountNotifier.java

```diff
@@ -244,6 +244,9 @@ private static String parsePrimaryBoss(String boss, String type) {
                 if (KillCountService.CG_NAME.equalsIgnoreCase(boss))
                     return KillCountService.CG_BOSS;
                 return null;
+            case "harvest":
+                if (KillCountService.HERBIBOAR.equalsIgnoreCase(boss))
+                return KillCountService.HERBIBOAR;
```

Comment:
Is this the best way for adjust the in-game's herbiboar to a capitalized Herbiboar?

Reply:
lgtm!